src/ServiceControl.Monitoring/ServiceControl.Monitoring.csproj

```diff
@@ -12,11 +12,26 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>ServiceControl.Monitoring</RootNamespace>
     <AssemblyName>ServiceControl.Monitoring</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
```

Comment:
core currently targets 4.5.2 and will move to 4.6.1 or higher with the next major.